products/apache-maven.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Apache maven
+layout: product
+category: build
+versionCommand: mvn --version
+permalink: /maven
+alternate_urls:
+-   /apache-maven
+-   /mvn
+releasePolicyLink: https://maven.apache.org/docs/history.html
+changelogTemplate: "https://github.com/apache/maven/releases/tag/v__LATEST__"
+auto:
+-   git: https://github.com/apache/maven.git
+# See https://rubular.com/r/stJ20etRIblK0J for reference
+    regex: 'https://rubular.com/r/hQMqEiUNHQOGZL'
+releases:
+-   releaseCycle: "3.8"
+    eol: false
+    support: 2023-11-24
```

Comment:
I do not see any end-of-support dates on https://maven.apache.org/docs/history.html. Could you add the link(s) containing that information in the description ?

Reply:
The only I got are the one for Maven 2 : 
![image](https://user-images.githubusercontent.com/5235127/204085409-48b5e27c-a1af-4d4e-993d-528f638ed59c.png)

Reply:
... and Maven 1 : 
![image](https://user-images.githubusercontent.com/5235127/204085421-a2abb1d4-1833-40b7-af46-427c3cde74a7.png)